js/views/itemEditVw.js

```diff
@@ -241,7 +261,7 @@ module.exports = Backbone.View.extend({
         dataURI = canvas.toDataURL('image/jpeg', 0.75);
         dataURI = dataURI.replace(/^data:image\/(png|jpeg);base64,/, "");
         imageList.push(dataURI);
-        if(i+1 === imageCount) {
+        if(loaded === imageCount) {
```

Comment:
Using a counter here rather than relying on the index of the loop, because I found that if the last image in the loop was not loaded last, any images loaded afterwards were not being uploaded.